.github/workflows/wheels.yml

```diff
@@ -34,35 +39,35 @@ jobs:
           python-version: '3.9'
 
       - uses: docker/setup-qemu-action@v1
-        if:  startsWith(matrix.os, 'ubuntu')
+        if: runner.os == 'Linux'
         name: Set up QEMU
 
       - name: Install prerequisites for macOS
-        if: contains(matrix.os, 'macos')
+        if: runner.os == 'macOS'
         env:
           # Best compatibility, even with older releases of macOS.
           MACOSX_DEPLOYMENT_TARGET: "10.9"
         run: |
-          brew install --build-from-source --no-binaries --force szip
-          brew reinstall --build-from-source --no-binaries --force c-blosc bzip2 hdf5 lz4 lzo snappy zstd zlib
+          brew reinstall --build-from-source --no-binaries --force  bzip2 lz4 lzo snappy zstd zlib
           brew link --overwrite --force bzip2 zlib
 
       - name: Install cibuildwheel
         run: |
           python -m pip install --upgrade cibuildwheel
 
-      - name: Build wheels for Linux or macOS (64-bit | aarch64)
+      - name: Build wheels for Linux or macOS (64-bit)
         run: |
           python -m cibuildwheel --output-dir wheelhouse
         env:
           CIBW_ARCHS_LINUX: ${{ matrix.arch }}
           CIBW_ARCHS_MACOS: ${{ matrix.arch == 'aarch64' && 'arm64' || 'x86_64'}}
           CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-*"
-          CIBW_BEFORE_ALL_LINUX: "yum -y update && yum install -y epel-release && yum install -y hdf5-devel zlib-devel bzip2-devel lzo-devel blosc-devel"
+          CIBW_BEFORE_ALL_LINUX: "yum -y update && yum install -y epel-release && yum install -y zlib-devel bzip2-devel lzo-devel ${{ matrix.arch == 'aarch64' && 'blosc-devel' || ''}} && ./ci/github/get_hdf5_if_needed.sh"
```

Comment:
As far as I can understand `${{ matrix.arch == 'aarch64' && 'blosc-devel' || ''}}` should be no longer necessary, correct?

Reply:
with the changes to master / the rebase i've just made, that's correct (also epel-release is no longer necessary).

Reply:
I agree, will polish after merge.